app/controllers/notification/AreYouADesignatedMemberOfTheLLPThatTheDisclosureWillBeAboutController.scala

```diff
@@ -85,4 +86,11 @@ class AreYouADesignatedMemberOfTheLLPThatTheDisclosureWillBeAboutController @Inj
       case _ => (Nil, false)
     }
   }
+
+  def isDisclosure(userAnswers: UserAnswers): Boolean = {
+    userAnswers.submissionType match {
+      case Disclosure => true
+      case _ => false
+    }
+  }
```

Comment:
Instead of having this method in every controller can we move it to a separate helper object?